### PR TITLE
Dependencies update

### LINKED ISSUE #22: Update eslint config
`@ackee/eslint-config@2.2.0`

### PR DESCRIPTION
**Pull request by jstorm31**
- Cypress updated
- petrus, antonio updated
- jerome and `translatableFactory` usage updated
- updated build scripts to correspond to the new Gitlab CI configuration
- resolved security vulnerabilities from audit

Closes #22

### CODE REVIEW

File: packages/cra-template/template.json [hunk 1]
```diff
@@ -34,20 +34,28 @@
...
-      "lokse": "^1.3.0",
+      "lokse": "^1.3.1",
       "fela-plugin-validator": "11.x",
       "fela-plugin-prefixer": "11.x",
       "gitmoji-changelog": "2.x",
       "husky": "4.x",
       "imagemin-lint-staged": "0.x",
       "lint-staged": "10.x"
     },
+    "resolutions": {
```
Comment (jstorm31) [inline packages/cra-template/template.json:45]: Isn't there a better way? E.g.update CRA packages or rebase patch version?

Comment (cermakjiri) [reply packages/cra-template/template.json:45]: - Let's do it this way only as a temp. solution until it's rebased to cra@4.x.
- Ye, the best practice is to rebase the latest version of original CRA and then possibly add custom resolutions.

**Review by cermakjiri — APPROVED** [review]

**Comment by cermakjiri** [comment]
https://travis-ci.com/github/AckeeCZ/create-react-app/jobs/480254613

File: packages/cra-template/template.json (hunk 1 above)
Comment (jstorm31) [reply packages/cra-template/template.json:45]: Ok, agree 👍

**Pull request merged**